packages/ui-toolkit/src/components/atoms/RadioButton/RadioButton.tsx

```diff
@@ -16,7 +16,8 @@ const RadioButton = (props: Props) => {
     onSelect,
     radioDirection,
     dataTestId,
-    isDisabled
+    isDisabled,
+    labelClassName
```

Comment:
Classname directly cannot be passed into components. This will over ride the mint restrictions.

Reply:
We need to sync, I have to understand the requirement. Posibly we would have a work around solution already.